app/util/files/FileControllerMixin.js

```diff
@@ -0,0 +1,151 @@
+﻿Ext.define('CpsiMapview.util.files.FileControllerMixin', {
+    extend: 'Ext.Mixin',
+    requires: ['CpsiMapview.util.files.Report'],
+
+    onAddFileClick: function (btn/*, e, eOpts*/) {
+        var grid = btn.up('grid');
+        var vm = this.getView().viewModel;
```

Comment:
I think you can use this.getViewModel() for clarity?

Reply:
done

Reply:
The branch needs to be pushed? I can't see any changes to the `var vm = this.getView().viewModel;` line.